nservicebus/errors/subscribing-to-error-notifications.md

```diff
@@ -17,7 +17,7 @@ Error notifications are available for several events.
 
 This API was added in Version 5.1.
 
-These event are exposed via the `BusNotifications` class that can be injected via DI.
+These event are exposed via the `BusNotifications` (NServiceBus Version 5) or `Notifications` (NServiceBus Version 6) class that can be injected via DI.
```

Comment:
See version language here: https://github.com/particular/docs.particular.net#version-language

> These event are exposed via the `BusNotifications` (NServiceBus Version 5) or `Notifications` (NServiceBus Version 6) class that can be injected via DI.

How about:
These events are exposed via the `BusNotifications` class for Versions 5 and below.  For Versions 6 and above, these events are exposed via the `Notifications` class. This class can be injected via DI.